modules/phase_field/src/kernels/ACInterfaceKobayashi1.C

```diff
@@ -39,69 +42,39 @@ ACInterfaceKobayashi1::ACInterfaceKobayashi1(const InputParameters & parameters)
   {
     _dLdarg[i] = &getMaterialPropertyDerivative<Real>("mob_name", _coupled_moose_vars[i]->name());
     _depsdarg[i] = &getMaterialPropertyDerivative<Real>("eps_name", _coupled_moose_vars[i]->name());
-    _ddepsdarg[i] = &getMaterialPropertyDerivative<Real>("eps1_name", _coupled_moose_vars[i]->name());
+    _ddepsdarg[i] = &getMaterialPropertyDerivative<Real>("deps_name", _coupled_moose_vars[i]->name());
   }
 }
 
 RealGradient
 ACInterfaceKobayashi1::precomputeQpResidual()
 {
-  // Set interfacial part of residual
-  RealVectorValue v = _eps[_qp]* _deps[_qp] * _L[_qp] * _grad_u[_qp];
-  std::swap(v(0), v(1));
-  v(0) *= -1.0;
-  return v;
+  // Set modified gradient vector
+  RealGradient v( - _grad_u[_qp](1), _grad_u[_qp](0), 0 );
```

Comment:
We don't put spaces on the interior of parens. And I would suggest making it `const`.